architect.yml

```diff
@@ -51,8 +51,7 @@ services:
     interfaces:
       main: &api-port 3000
     liveness_probe:
-      path: /users
-      port: *api-port
+      command: curl localhost:3000/users
```

Comment:
Does this fail correctly?

I think it needs to be:
`curl --fail http://localhost:3000/users || exit 1`

Reply:
it does now